src/utils/filteredTodos.tsx

```diff
@@ -0,0 +1,22 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export const FilteredTodos = (todos: Todo[], filterBy: Status): Todo[] => {
+  let filteredTodos = [...todos];
```

Comment:
You can try do not use let in this case. 
Work with array and return method result.
You don't need reassign in this case.